### DISCUSSION TITLE
2021.10.07 회의록

### DISCUSSION THREAD

**Discussion by kwakjeeyoon** [boostcampaitech2/object-detection-level2-cv-06#148] [General]
- 해상도 : (1024,1024)
- learing rate 은 모델별로 수렴하는 값이 다르니까
- epoch 단위로

- 재환 : ssd는 모델 자체도 크고 성능도 좋지 않아서 중단 → 성능을 높이는 다른 기술을 찾아보는 걸로
- 지윤 : universenet pretrained 버전으로 돌리고 있다. → 다양한 크기의 bbox를 찾는데 특화된 모델이라고 하는데 왜 그런지는 논문 읽고 정리해서 올리도록
- 경재 : validation loss 가 정체되는 부분을 찾아야겠다.
- 나은 : Yolo epoch 더 크게 해서 돌리고 있고 EfficientDet 코드 보고 있다. 추가해서 Yolox, YoloF 버전을 돌려볼 예정
- 지유 : 모델의 구조를 살펴보고 있다. 정리 사항은 노션에 올려놓겠다.
- 준혁 : detr과 cbnet 보고 있다. detr은 learning rate을 높이고 epoch을 줄이는 방법은 성능이 안좋다. 모자이크를 각 모델마다 적용하기 어려울 것 같아서 **모자이크 데이터셋**을 만드는 식으로 script를 만들고 있다.
